joomla/joomla-cms
# [4.4] Bootstrap 5.3

### Summary of Changes
Upgrades to bootstrap 5.3. This isn't 100% b/c but its as b/c as I can make it. The biggest issue is that bootstrap introduced a `--link-color` parameter that a) causes various parts of our code to not compile and b) does the same thing as our `template-link-color`/`casseopia-color-link` EVs. I've tried to migrate our usage to the new BS variable - but also keep us setting the old variables for templates that depend on them.

Note as this is a first pass I'm deliberately disabling dark mode support *for now*. This should be enabled but will require another pass over our SCSS and I'd rather just get the dependency upgrade in for now before enabling the shiny new features.

### Testing Instructions
Check core loads and styling works as expected in both frontend and backend. If you have any extensions to hand ideally check that their styling also continue to work.

### Link to documentations
Please select:
